client/lib/validation.rs

```diff
@@ -144,7 +144,7 @@ pub(crate) fn validate_query_response(
         let proof_str = proof
             .as_str()
             .ok_or(ValidateResponseError::ValidateResponseFailedToParse)?;
-        let proof_bytes = hex::decode(proof_str)
+        let proof_bytes = checksummed_hex::decode(proof_str)
```

Comment:
```suggestion
        let proof_bytes = base16::decode(proof_str)
```

Reply:
Fixed in [e93dc37](https://github.com/nilz3ro/casper-node/pull/5/commits/e93dc3787a9983c352ebfdba0fc6bccb8f4d7c41).